### DISCUSSION TITLE
Ctrl-Backspace to delete word

### DISCUSSION THREAD

**Discussion by mashtonian** [d99kris/nchat#316] [Ideas]
To make the post editing experience more consistent with other programs, could ctrl-backspace (and possiblty ctrl-w) be used to delete back a whole word?

**Comment by mashtonian** [comment]
ah, just realised that ctrl-w is already the default binding for another action. Just ctrl-backspace then.

**Comment by d99kris** [comment]
One can configure `backward_kill_word` (delete previous word) in `key.conf` to customize which key binding to use. By default nchat maps this to `alt/opt-backspace`.

**Comment by d99kris** [comment]
I will proceed to close this discussion for now. Feel free to re-open if you have any follow-up questions or ideas.
